plugins/amqp/alerta_amqp.py

```diff
@@ -19,7 +19,6 @@
 
 AMQP_URL = os.environ.get('REDIS_URL') or os.environ.get('AMQP_URL') or app.config.get('AMQP_URL', DEFAULT_AMQP_URL)
 AMQP_TOPIC = os.environ.get('AMQP_TOPIC') or app.config.get('AMQP_TOPIC', DEFAULT_AMQP_TOPIC)
```

Comment:
Can you use `get_config()` on these two variables as well? It would be confusing if only some were converted to use dynamic config.

Reply:
The problem is that those two variables are used in __init__, and the get_config relies on **kwargs to get the config from the app, or it will only use the os.environ to check that it is defined.
May I use the "app" reference in __init__ as **kwargs ? Like this :
`get_config('AMQP_URL', default=DEFAULT_AMQP_URL, type=str, app)`
I am a little confused by what **kwargs is supposed to be...

Reply:
That's fine. By it's nature plugin "inits" aren't dynamic.